drivers/SmartThings/matter-switch/profiles/light-level-colorTemperature-2700k-6500k.yml

```diff
@@ -0,0 +1,26 @@
+name: light-level-colorTemperature-2700k-6500k
```

Comment:
This profile is a duplicate of the profile `light-color-level-2700K-6500K` that was introduced here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/872/files

Please delete this file and then update your repo to include the latest changes, then point your fingerprint to the duplicate profile that already exists. Then this should be good to go!